bitcoin/bitcoind/entrypoint.sh

```diff
@@ -20,8 +20,13 @@ sleep 10
 
 alias btccli='./bitcoin-cli -regtest -datadir=datadir -rpcport=18332 -rpcuser=user -rpcpassword=password'
 
-# Get an address.
-address=$(btccli getnewaddress)
+# Use a specific private key.
+privateKey="cTj6Z9fxMr4pzfpUhiN8KssVzZjgQz9zFCfh87UrH8ZLjh3hGZKF"
```

Comment:
This is not necessary but it can be if there is such intention. `bitcoin/bitcoind/entrypoint.sh` is the local BTC node script and here we use an explicit private key in order to send the coinbase to a predictable address and use it as input for deposit transactions in the e2e test script executed against the local environment.